sentinel/pkg/checker/event/app.go

```diff
@@ -15,6 +15,7 @@ import (
 
 const AlarmOffset = 3
 const VRF_EVENT = "vrf_random_words_fulfilled"
+const MAX_FEED_SUBMISSION_COUNT = 8
```

Comment:
rather than having this as const variable, how about calculating the value from submission interval? every feeds has expected intervals, and I think it'd be nice if this max count value can be relatively set based on each pairs' expected submission interval. 

later, we might have price pairs with different submission intervals

Reply:
@nick-bisonai thanks for the suggestion. I made that variable dynamic, pls take a look